app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCapturePagerAdapter.java

```diff
@@ -40,11 +42,28 @@ public EventCapturePagerAdapter(FragmentActivity fragmentActivity,
         this.eventUid = eventUid;
         pages = new ArrayList<>();
         pages.add(EventPageType.DATA_ENTRY);
-        pages.add(EventPageType.ANALYTICS);
-        pages.add(EventPageType.RELATIONSHIPS);
+
+        if (displayAnalyticScreen) {
+            pages.add(EventPageType.ANALYTICS);
+        }
+
+        if (displayRelationshipScreen) {
+            pages.add(EventPageType.RELATIONSHIPS);
+        }
```

Comment:
@Balcan This fixed  ANDROAPP-4464 but create the ANDROAPP-4506 animation glitch

Reply:
Yes @ferdyrod . The animation problem happened because we had in the pages relationships and analytics (this was removed in dashboard)

Reply:
great!